devops/scripts/benchmarks/main.py

```diff
@@ -590,6 +599,16 @@ def validate_and_parse_env_args(env_args):
     options.build_jobs = args.build_jobs
     options.hip_arch = args.hip_arch
 
+    # Set the log level
+    log_level = (
+        LogLevel.DEBUG.value
+        if args.verbose
+        else LogLevel.from_string(args.log_level).value
+    )
+    log.logger.setLevel(log_level)
+    for handler in log.logger.handlers:
+        handler.setLevel(log_level)
```

Comment:
I feel like this logic should be hidden inside of the logger class.

Reply:
Refactored